docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -455,6 +455,7 @@ code: |
   metadata_code = interview.blocks.appendObject()
   metadata_code.template_key = "al metadata"
   metadata_code.data = {
+    "al_weaver_version": package_version_number,
```

Comment:
This is good, but gonna use it as a starting point: do we want to try to determine between version bumps, i.e. git commit hash of the current branch instead? I feel that would only be useful for devs and would be a lot more complicated than this, so maybe it's not useful.

Reply:
Would be neat but much more complicated. Not sure if this is currently exposed to a running interview